client/verta/verta/runtime.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+
+import json
+from typing import Any, Dict
+
+
+class LogHandler:
+    """
+    Container class for collecting and dumping log entries.
+    """
+    def __init__(self):
+        self.attributes: Dict[str, Any] = dict()
+
+    def __repr__(self):
+        return str(self.attributes)
```

Comment:
Not sure about this, but maybe

```suggestion
        return repr(self.attributes)
```

?

Reply:
Refactored out